openedx_learning/apps/authoring/publishing/api.py

```diff
@@ -457,13 +473,199 @@ def set_draft_version(
     edited some content). Setting a Draft's version to None is like deleting it
     from Studio's editing point of view (see ``soft_delete_draft`` for more
     details).
+
+    Calling this function attaches a new DraftChange and attaches it to a
```

Comment:
Self note: search-replace old references to `DraftChange` (it's `DraftChangeLogRecord` now, for consistency with the publishing side).